### PR TITLE
Remove draw feature as Android (QC board) doesn't support video playback

### PR DESCRIPTION
**Pull request by wiperz1789**

### CODE REVIEW

File: vision/shared/PlottingImageListener.h [hunk 1]
```diff
@@ -142,7 +142,7 @@ class PlottingImageListener : public vision::ImageListener, public PlottingListe
             viz_.drawPoints(f.getFacePoints());
 
             // Draw a face on screen
-            viz_.drawFaceMetrics(f, bbox, draw_face_id_);
+            viz_.drawFaceMetrics(f, bbox);
```
Comment (adennie) [inline vision/shared/PlottingImageListener.h:145]: seems like we don't need the Visualizer any more.

Comment (wiperz1789) [reply vision/shared/PlottingImageListener.h:145]: No we still need it as it's used in output to csv file

Comment (adennie) [reply vision/shared/PlottingImageListener.h:145]: The only part of Visualizer that's used by PlottingImageListener is the label information like EMOTIONS, EXPRESSIONS, MOODS, DOMINANT_EMOTIONS, etc.  You could just move those to PlottingListener.h.

File: vision/shared/PlottingImageListener.h [hunk 2]
```diff
@@ -145,7 +145,7 @@ class PlottingImageListener : public vision::ImageListener, public PlottingListe
             viz_.drawFaceMetrics(f, bbox, draw_face_id_);
         }
 
-        viz_.showImage();
+        // viz_.showImage();
```
Comment (umangmehta12) [inline vision/shared/PlottingImageListener.h:148]: probably remove this comment

File: vision/shared/PlottingOccupantListener.h [hunk 3]
```diff
@@ -88,7 +88,7 @@ class PlottingOccupantListener : public vision::OccupantListener, public Plottin
             }
         }
 
-        viz_.showImage();
+        //viz_.showImage();
```
Comment (umangmehta12) [inline vision/shared/PlottingOccupantListener.h:91]: here as well

**Comment by wiperz1789** [comment]
> Looks good to me, I'm not sure but if the intention of commenting `viz_.showImage` is for customers to un-comment it as needed maybe document it somewhere on how they could see the UI if not remove it.

I can do either of this! what do you(@umangmehta12 ) and @adennie  suggest?
removing the comments sounds better to me.

**Comment by adennie** [comment]
removing is good.

**Pull request merged**